### PR TITLE
Add stylelint-suitcss to plugin list

### PR DESCRIPTION
**Pull request by simonsmith**
Also updates deprecation notices in READMEs and code

Addresses #2226

https://github.com/suitcss/stylelint-suitcss

### CODE REVIEW

File: lib/rules/custom-property-no-outside-root/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # custom-property-no-outside-root
 
-***Deprecated: See [CHANGELOG](../../../CHANGELOG.md).***
+***Deprecated: See [`stylelint-suitcss`](https://github.com/suitcss/stylelint-suitcss)***
```
Comment (m-allanson) [inline lib/rules/custom-property-no-outside-root/README.md]: Can you change these README.md notices to:

```
***Deprecated: instead use the community [`stylelint-suitcss`](https://github.com/suitcss/stylelint-suitcss) plugin pack***
```

File: lib/rules/custom-property-no-outside-root/index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ const rule = function (actual) {
     }
 
     result.warn((
-      `'${ruleName}' has been deprecated and in 8.0 will be removed.`
+      `'${ruleName}' has been deprecated and in 8.0 will be removed. Use 'stylelint-suitcss instead'`
```
Comment (m-allanson) [inline lib/rules/custom-property-no-outside-root/index.js]: Can you change all the result updates from `Use 'stylelint-suitcss instead'` to `Instead use the community 'stylelint-suitcss' plugin pack.`?

**Comment by simonsmith** [comment]
@m-allanson Changes made and rebased onto master 👍

**Comment by davidtheclark** [comment]
Thanks @simonsmith!

**Pull request merged**